configs/Penguin-XeonPlatinum-OmniPath/system_definition.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+system_definition:
+  name: Penguin-XeonPlatinum-OmniPath
+  manufacturer:
+    vendor: PenguinComputing
+    name: RelionCluster
+  processor:
+    vendor: Intel
+    name: XeonPlatinum924248C
+    ISA: x86_64
+  accelerator:
+    vendor: ""
+    name: ""
+    GPU_ISA: ""
```

Comment:
Is this better than having `accelerator: None`?

I know the example is contrived, but it would distinguish more clearly between no accelerator and a weirdly named one.

Reply:
My thought was that I will want to make a table of these system descriptions, and each line is a row - which is why I kept all of the rows.  Is that reasonable?

I am happy to have None instead of the "" if that makes it cleaner.